nf_core/utils.py

```diff
@@ -681,56 +682,22 @@ def get_biocontainer_tag(package, version):
         A ValueError, if the package name can not be found (404)
     """
 
-    biocontainers_api_url = f"https://api.biocontainers.pro/ga4gh/trs/v2/tools/{package}/versions/{package}-{version}"
-
-    def get_tag_date(tag_date):
-        """
-        Format a date given by the biocontainers API
-        Given format: '2021-03-25T08:53:00Z'
-        """
-        return datetime.datetime.strptime(tag_date, "%Y-%m-%dT%H:%M:%SZ")
+    quay_api_url = f"{QUAY_REPOSITORY_API_ENDPOINT}/biocontainers/{package}/tag"
 
     try:
-        response = requests.get(biocontainers_api_url)
+        response = requests.get(quay_api_url, params={"onlyActiveTags": True, "limit": 1})
     except requests.exceptions.ConnectionError:
-        raise LookupError("Could not connect to biocontainers.pro API")
+        raise LookupError("Could not connect to quay.io API")
     else:
         if response.status_code == 200:
             try:
-                images = response.json()["images"]
-                singularity_image = None
-                docker_image = None
-                all_docker = {}
-                all_singularity = {}
-                for img in images:
-                    # Get all Docker and Singularity images
-                    if img["image_type"] == "Docker":
-                        # Obtain version and build
-                        match = re.search(r"(?::)+([A-Za-z\d\-_.]+)", img["image_name"])
-                        if match is not None:
-                            all_docker[match.group(1)] = {"date": get_tag_date(img["updated"]), "image": img}
-                    elif img["image_type"] == "Singularity":
-                        # Obtain version and build
-                        match = re.search(r"(?::)+([A-Za-z\d\-_.]+)", img["image_name"])
-                        if match is not None:
-                            all_singularity[match.group(1)] = {"date": get_tag_date(img["updated"]), "image": img}
-                # Obtain common builds from Docker and Singularity images
-                common_keys = list(all_docker.keys() & all_singularity.keys())
-                current_date = None
-                for k in common_keys:
-                    # Get the most recent common image
-                    date = max(all_docker[k]["date"], all_docker[k]["date"])
-                    if docker_image is None or current_date < date:
-                        docker_image = all_docker[k]["image"]
-                        singularity_image = all_singularity[k]["image"]
-                        current_date = date
-                return docker_image["image_name"], singularity_image["image_name"]
+                return response.json()["tags"][0]["name"]
```

Comment:
I tried with hla-la and I get an older version than for bioconda. Should we check if versions match and downgrade bioconda if they don't?
```
conda (params.enable_conda ? "bioconda::hla-la=1.0.3" : null)
def container_image = "hla-la:1.0.1--h1b026d1_5"
container [ params.container_registry ?: 'quay.io/biocontainers' , container_image ].join('/')
```

Reply:
Can you manually check if there's a matching hla-la version? I was just lazy and hoped asking it for only one image gave you the latest :grimacing: 

I can check that myself.

Reply:
Okay there is a 1.0.3 version, the problem is it got pushed 4 months ago, but the 1.0.1 version with the `_5` build got pushed 3 months ago.

Reply:
Oh, I see that's an edge case. Then I guess we can let the user decide, but maybe still print a warning if the versions are different.